lib/membrane_flv_plugin/demuxer.ex

```diff
@@ -175,9 +172,10 @@ defmodule Membrane.FLV.Demuxer do
           {[
              stream_format:
                {pad,
-                %Membrane.H264.RemoteStream{
+                %H264{
                   alignment: :au,
-                  decoder_configuration_record: packet.payload
+                  # TODO can this be avc1
+                  stream_structure: {:avc3, packet.payload}
```

Comment:
good question, cc @varsill

Reply:
since flv is used for livestreaming it seems to allow for parameter sets in-band, therefore I would stick to `:avc3`